Only pass thru specific keys that are known rather than whatever is set in
the URI.

https://issuetracker.google.com/260531249

--- a/mosh_app/mosh_window.js
+++ b/mosh_app/mosh_window.js
@@ -99,9 +99,25 @@ mosh.CommandInstance.prototype.run = function() {
        'height: 0;');
   this.moshNaCl_.setAttribute('src', nacl_nmf_file);
   this.moshNaCl_.setAttribute('type', nacl_mime_type);
-  for (var k in this.args_) {
-    this.moshNaCl_.setAttribute(k, this.args_[k]);
-  }
+  // Only pass down specific values from the URI argv.  See b/260531249.
+  [
+    'addr',
+    'dns-resolver',
+    'family',
+    'key',
+    'mode',
+    'mosh-escape-key',
+    'port',
+    'remote-command',
+    'server-command',
+    'trust-sshfp',
+    'use-agent',
+    'user',
+  ].forEach((k) => {
+    if (this.args_[k] !== undefined) {
+      this.moshNaCl_.setAttribute(k, this.args_[k]);
+    }
+  });
 
   // Output special text (e.g., ANSI escape sequences) if desired.
   chrome.storage.sync.get('term_init_string', function(o) {
